wpeverest/user-registration
# UR-1276 Tweak - Default form fields arrangement

### All Submissions:






### Changes proposed in this Pull Request:
Our default form appears in two columns. Please check the screenshot. The way the current default form uses the column is wrong. We should have instead created two rows each with two columns. This PR provides this tweak. 


### How to test the changes in this Pull Request:

1. The Default Form created during the fresh installation of our plugin should have this tweak.
2. Create a new form and select Start From Scratch option and the form created should have this tweak.
3. Form created from both ways must have the Username and User email field in a single row with two columns and Password and Confirm Password fields must be in another row with two columns.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Default form fields arrangement
